tests/manager/manager-debug-001.phpt

```diff
@@ -29,7 +29,7 @@ foreach($content as $line) {
 ===DONE===
 <?php exit(0); ?>
 --EXPECTF--
-[%s]     PHONGO: DEBUG   > Creating Manager, phongo-0.%d.%d[%s] - mongoc-1.%s, libbson-1.%s
+[%s]     PHONGO: DEBUG   > Creating Manager, phongo-%d.%d.%d[%s] - mongoc-1.%s, libbson-1.%s
```

Comment:
I'd imagine hhvm would need to skip debug messaging like this, considering the trace will likely be subtly different.
I don't have a preference for replacing 'phongo' with %s or not